### PR TITLE
Doc updates for SDK 3.10

### PR DESCRIPTION
**Pull request by mhelf-dolby**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+
+- Introduced a new [preview](https://api-references.dolby.io/comms-sdk-flutter/dolbyio_comms_sdk_flutter/LocalAudio/preview.html) method that allows the local participant to test different capture modes and voice fonts before a conference. The method sets a preview recorder that records the participant's audio and plays it back. Before playing the recorded audio, set the [captureMode](https://api-references.dolby.io/comms-sdk-flutter/dolbyio_comms_sdk_flutter/LocalAudio/setCaptureMode.html) to a preferred setting that you wish to try.
```
Comment (graduad) [inline CHANGELOG.md:9]: I am not sure this is a method. I see it more as a service.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+
+- Introduced a new [preview](https://api-references.dolby.io/comms-sdk-flutter/dolbyio_comms_sdk_flutter/LocalAudio/preview.html) service that allows the local participant to test different capture modes and voice fonts before a conference. The service sets a preview recorder that records the participant's audio and plays it back. Before playing the recorded audio, set the [captureMode](https://api-references.dolby.io/comms-sdk-flutter/dolbyio_comms_sdk_flutter/LocalAudio/setCaptureMode.html) to a preferred setting that you wish to try.
```
Comment (graduad) [inline CHANGELOG.md:9]: Perhaps 

`The service records the local participant's audio and plays it back thereafter.`

it's a bit more accurate then

`The service sets a preview recorder that records the participant's audio and plays it back.`

Also perhaps

`Before recording the audio for previewing, set the ...`

in stead of:

`Before playing the recorded audio, set the ...`

Comment (mhelf-dolby) [reply CHANGELOG.md:9]: Are you sure? I think that Yuriy explained to me some time ago that you record your natural voice and then, before playing it, you can change it using setCaptureMode.

Comment (graduad) [reply CHANGELOG.md]: The flow we use for testing is:
1. select voice font
2. start recording
3. (optional) cancel 
4. play recording
5. release recording

Also to change a bit the test I wrote:

`The service records the local participant's audio and allows the recording to be played back thereafter.`

This in stead of:

`The service records the local participant's audio and plays it back thereafter.`

Comment (graduad) [reply CHANGELOG.md]: Ok. Just tested agin. It works as you explained as well.

Comment (graduad) [reply CHANGELOG.md]: Since is works both ways we can leave it as is.

**Review by graduad — APPROVED** [review]

**Pull request merged**